packages/appreader/webpack.config.ts

```diff
@@ -60,6 +60,11 @@ const config: Configuration = {
   ],
   resolve: {
     extensions: ['.tsx', '.ts', '.js'],
+    fallback: {
```

Comment:
Not sure what this one does?

Reply:
It's related to the Next 13 upgrade. `@omnivore/appreader` imports a package from `@omnivore/web` and throws an error unless I fallback the imports.

```
$ webpack --mode production
Webpack Bundle Analyzer saved report to /Users/luke/Developer/omnivore/packages/appreader/build/report.html
assets by status 1.28 MiB [cached] 1 asset
orphan modules 7.36 MiB [orphan] 1521 modules
runtime modules 2.37 KiB 7 modules
cacheable modules 3.17 MiB
  modules by path ../../node_modules/ 1.23 MiB 225 modules
  modules by path ../web/ 240 KiB
    modules by path ../web/node_modules/markdown-it/lib/ 166 KiB 53 modules
    modules by path ../web/styles/*.css 22 KiB 2 modules
    modules by path ../web/node_modules/linkify-it/ 23.9 KiB 2 modules
    + 2 modules
  modules by path ./ 1.71 MiB
    modules by path ./node_modules/style-loader/dist/runtime/*.js 5.75 KiB 6 modules
    modules by path ./node_modules/css-loader/dist/runtime/*.js 2.33 KiB
      ./node_modules/css-loader/dist/runtime/noSourceMaps.js 64 bytes [built] [code generated]
      ./node_modules/css-loader/dist/runtime/api.js 2.26 KiB [built] [code generated]
    ./src/index.jsx + 412 modules 1.7 MiB [built] [code generated]

ERROR in ../../node_modules/next/dist/compiled/gzip-size/index.js 1:2832-2845
Module not found: Error: Can't resolve 'fs' in '/Users/luke/Developer/omnivore/node_modules/next/dist/compiled/gzip-size'
 @ ../../node_modules/next/dist/shared/lib/bloom-filter.js 42:33-77
 @ ../../node_modules/next/dist/shared/lib/router/router.js 1649:36-69
 @ ../../node_modules/next/dist/client/router.js 42:57-95
 @ ../../node_modules/next/router.js 1:0-48
 @ ../web/components/templates/article/SetLabelsControl.tsx 35:0-40 279:15-24
 @ ../web/components/templates/article/SetLabelsModal.tsx 28:0-54 252:32-48
 @ ../web/components/templates/article/SetLabelsModalPresenter.tsx 15:0-50 39:27-41 60:27-41
 @ ../web/components/templates/article/HighlightsLayer.tsx 41:0-77 1106:42-74
 @ ../web/components/templates/article/ArticleContainer.tsx 22:0-74 472:26-41
 @ ./src/index.jsx 12:0-95 108:36-52

ERROR in ../../node_modules/next/dist/compiled/gzip-size/index.js 1:2878-2895
Module not found: Error: Can't resolve 'stream' in '/Users/luke/Developer/omnivore/node_modules/next/dist/compiled/gzip-size'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "stream": require.resolve("stream-browserify") }'
        - install 'stream-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "stream": false }
 @ ../../node_modules/next/dist/shared/lib/bloom-filter.js 42:33-77
 @ ../../node_modules/next/dist/shared/lib/router/router.js 1649:36-69
 @ ../../node_modules/next/dist/client/router.js 42:57-95
 @ ../../node_modules/next/router.js 1:0-48
 @ ../web/components/templates/article/SetLabelsControl.tsx 35:0-40 279:15-24
 @ ../web/components/templates/article/SetLabelsModal.tsx 28:0-54 252:32-48
 @ ../web/components/templates/article/SetLabelsModalPresenter.tsx 15:0-50 39:27-41 60:27-41
 @ ../web/components/templates/article/HighlightsLayer.tsx 41:0-77 1106:42-74
 @ ../web/components/templates/article/ArticleContainer.tsx 22:0-74 472:26-41
 @ ./src/index.jsx 12:0-95 108:36-52

ERROR in ../../node_modules/next/dist/compiled/gzip-size/index.js 1:2928-2943
Module not found: Error: Can't resolve 'zlib' in '/Users/luke/Developer/omnivore/node_modules/next/dist/compiled/gzip-size'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "zlib": require.resolve("browserify-zlib") }'
        - install 'browserify-zlib'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "zlib": false }
 @ ../../node_modules/next/dist/shared/lib/bloom-filter.js 42:33-77
 @ ../../node_modules/next/dist/shared/lib/router/router.js 1649:36-69
 @ ../../node_modules/next/dist/client/router.js 42:57-95
 @ ../../node_modules/next/router.js 1:0-48
 @ ../web/components/templates/article/SetLabelsControl.tsx 35:0-40 279:15-24
 @ ../web/components/templates/article/SetLabelsModal.tsx 28:0-54 252:32-48
 @ ../web/components/templates/article/SetLabelsModalPresenter.tsx 15:0-50 39:27-41 60:27-41
 @ ../web/components/templates/article/HighlightsLayer.tsx 41:0-77 1106:42-74
 @ ../web/components/templates/article/ArticleContainer.tsx 22:0-74 472:26-41
 @ ./src/index.jsx 12:0-95 108:36-52

3 errors have detailed information that is not shown.
Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.

webpack 5.70.0 compiled with 3 errors in 8008 ms
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command
```